capitoli/Primo_Passo.rst

```diff
@@ -0,0 +1,235 @@
+
+.. activecode:: somma
+   :coach:
+   :caption: 
+
+   print(2+3)
+  
+
+Vediamo altri esempi:
+
+.. activecode:: differenza
+   :coach:
+   :caption: 
+  
+   print(15-11)
+
+.. activecode:: moltiplicazione
+   :coach:
+   :caption: 
+  
+   print(2*3)
+
+.. activecode:: divisione
+   :coach:
+   :caption: 
+  
+   print(15/5)
+
+Quindi il simbolo **+** è il comando per eseguire una somma;
+il simbolo **–** è l’ordine di eseguire una sottrazione; 
+il simbolo ***** indica una moltiplicazione;
+i simboli **/** e **//** indicano la divisione.
+Ora impariamo anche l’uso del simbolo che viene 
+chiamato “**elevato a**” 
+e che ha il significato mostrato 
+nei seguenti esempi:
+
+**PRIMO ESEMPIO**
+
+.. activecode:: primo esmpio
+   :coach:
+   :caption: ossia 2*2*2,tre volte
+  
+   print(2**3)
+
+**SECONDO ESEMPIO**
+
+.. activecode:: secondo esempio
+   :coach:
+   :caption: 
+  
+   print(2**4)
+   
+ossia 2*2*2*2,quattro volte
+
+Adesso prova tu ad esercitarti usando Python. Prova a scrivere:
+
+1.>>>10**2
+
+2.>>>10**3
+
+3.>>>3**2
+
+4.>>>3**3
+
+Come anticipato, possiamo usare due simboli diversi per la divisione. 
+Ai due simboli diversi corrispondono operazioni un po’ diverse, come mostrato 
+nei seguenti esempi:
+
+(e poi dai invio) otterrai:
+
+.. activecode:: esempio1
+   :coach:
+   :caption: divisione con /
+
+   .."todo: non vanno i decimali"
+   
+   print(5/2)
+ 
+verifica ancora:
+ 
+.. activecode:: esempio2
+   :coach:
+   :caption: divisione con /
+
+   
+   .."todo: non vanno i decimali"
+   
+   print(15/4)
+
+Se invece scriviamo: 
+
+.. activecode:: esempio3
+   :coach:
+   :caption: divisione con //
+   
+   print(5//2)
+   
+   print(5.0//2.0)
+
+otteniamo la parte intera della divisione.
+
+In sintesi, Python  possiede due divisioni che si 
+ottengono con i due simboli / e //. La prima divisione 
+genera numeri con virgola, la seconda divisione 
+produce la parte intera del risultato.
+
+Se in una divisione ti interessa solo sapere il resto usa il simbolo %: 
+
+.. activecode:: esempio
+   :coach:
+   :caption: divisione con %
+   
+   print(15%12)
+
+**Fai attenzione**. Gli americani, quando usano i numeri decimali, usano il punto 
+(.) e non la virgola. Anche Python segue questo sistema.
+
+Si possono scrivere tante operazioni una dopo l’altra che 
+verranno eseguite nell’ordine con cui sono scritte, con qualche 
+differenza. Infatti prima bisogna eseguire  gli ** e poi i * o / e 
+in ultimo i + o i -. **Di seguito alcuni esempi**.
+
+.. activecode:: operazioni1
+   :coach:
+   :caption: serie di operazioni
+   
+   print(2*3+4)
+
+.. activecode:: operaioni2
+   :coach:
+   :caption: serie di operazioni
+   
+   print(6/2-1)
+
+.. activecode:: operazioni3
+   :coach:
+   :caption: serie di operazioni
+   
+   print(10+3*2)
+
+(Infatti il calcolatore prima calcola 3*2 =6 e poi calcola 10+6)
+
+.. activecode:: operazioni4
+   :coach:
+   :caption: serie di operazioni
+   
+   print(10+3*2**3)
```

Comment:
Anche in questo caso, secondo me ci sono troppi esempi che rendono la lettura difficile.
Ne metterei 1/2 e poi gli altri li metterei come blocco di testo.